docs/examples/upscale-expanded-particles.ipynb

```diff
@@ -0,0 +1,203 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Revert downsampled, symmetry expanded particles\n",
+    "\n",
+    "Re-extracting particles after symmetry expansion is wasteful, since it creates a particle stack N times larger than is necessary, where N is the number of symmetry-related positions. However, it is often helpful during particle cleanup to work with smaller images, since most cleanup operations do not require high resolution information.\n",
```

Comment:
Rearrange suggestion

```suggestion
    "Re-extracting particles after symmetry expansion is wasteful, since it creates a particle stack N times larger than is necessary, where N is the number of symmetry-related positions. However, during particle cleanup it is often helpful to work with smaller images, since most cleanup operations do not require high resolution information.\n",
```